client/app/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let throwcast = angular.module('throwcast', [
```

Comment:
Since we aren't transpiling the client. Lets use `var` everywhere on the client code.